Tests/AsyncAwait+Tests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  AsyncAwait+Tests.swift
+//  NetworkingTests
+//
+//  Created by Ashli Rankin on 4/26/23.
+//  Copyright © 2023 Lickability. All rights reserved.
+//
+
+import XCTest
+@testable import Networking
+
+final class AsyncAwait_Tests: XCTestCase {
+
+    private let controller = NetworkController()
+    
+    func testSendMethod() async throws {
+        do {
+            let photos: [Photo] = try await controller.send(PhotoRequest.photosList)
```

Comment:
Also, since this is more than you signed up for, it'd be okay to me if you wanted to run this test to make sure it passes (which I’d imagine you’ve already done), remove it, and then we can work together next Engineering Time on mocking networking behavior to effectively test these kinds of APIs.

Reply:
I can definitely do that. I'd definitely like to learn how to mock our object so we can unit-test the validity of our code.

Reply:
Sounds good!

Reply:
I added a engineering time [idea here](https://www.notion.so/lickability/Mocking-networking-behavior-934d256531934fce91bb1ff212335c11?pvs=4)

Reply:
Thanks!